Update README.md (#45)
stuvusIT/xen_vman: README.md

Mount or umount the root filesystem from the given VM. For example `systemd start vm_mount@stuvus-web02` mounts the root filesystem of _web01 @stuvus_ into `/mnt/vms/stuvus-web01`. The mount point is automated created and removed.

## nfs vms

> All nfs exports need to be accessible on the hypervisor and for the accordingly vm. The exported path for the root filesystem must be following schema:

Reviewer: A few grammatical errors, see my suggestion:

```suggestion
All NFS exports need to be accessible for the hypervisor as well as the respective VM.
The exported path for the root filesystem must be of the following scheme:
```

> `{{ xen_vman_nfsroot_base }}/<vm name>/<vm name>-root` .

Reviewer: The middle part should be `<org name>`
```suggestion
`{{ xen_vman_nfsroot_base }}/<org name>/<vm name>-root` .
```
